packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -908,8 +909,21 @@ export default class TradeStore extends BaseStore {
                             const is_digit_contract = isDigitContractType(category?.toUpperCase() ?? '');
                             const is_multiplier = isMultiplierContract(category);
                             const contract_type = category?.toUpperCase();
+                            const is_call = category === 'Call';
```

Comment:
![image](https://github.com/binary-com/deriv-app/assets/56330681/3295b6c6-86b6-4dfb-94c7-7aa1ca1a6ab1)
Seems like the category should be in caps

Reply:
Also, can we avoid hard-coding? Instead we can use it as `CONTRACT_TYPES.CALL`
<img width="398" alt="image" src="https://github.com/binary-com/deriv-app/assets/56330681/bdea5165-e871-48d1-844b-37cb6bc3fba8">